include/nc4internal.h

```diff
@@ -210,9 +210,12 @@ typedef struct NC_VAR_INFO
     nc_bool_t dimscale;          /**< True if var is a dimscale */
     nc_bool_t *dimscale_attached;  /**< Array of flags that are true if dimscale is attached for that dim index */
     nc_bool_t deflate;           /**< True if var has deflate filter applied */
-    int deflate_level;
+    int deflate_level;           /**< If deflate is true, this is the deflate level, between 1 and 9. */
```

Comment:
dvar.c says valid range is 0 to 9 with 0 meaning no compression.  There was some discussion awhile ago that using 0 results in different behavior than not enabling `deflate` since it still runs through other logic even though there will be no compression.